packages/hoprd/rest-api-v3-spec.yaml

```diff
@@ -291,9 +291,6 @@ components:
       properties:
         channelId:
           $ref: '#/components/schemas/ChannelId'
-        challenge:
```

Comment:
Why did you this file?

Reply:
That was before your changes. Rebased and adapted it now.